src/components/molecules/Visualizer/Engine/Cesium/common.ts

```diff
@@ -287,31 +288,38 @@ export const animateFOV = ({
     typeof camera.frustum.fov === "number" &&
     camera.frustum.fov !== fov
   ) {
-    const fromFov = camera.frustum.fov;
-    return tweenInterval(
-      t => {
-        if (!(camera.frustum instanceof PerspectiveFrustum)) return;
-        camera.frustum.fov = (fov - fromFov) * t + fromFov;
-      },
-      easing || "inOutCubic",
-      (duration ?? 0) * 1000,
-    );
+    if (duration === 0) {
+      camera.frustum.fov = fov;
+    } else {
+      const fromFov = camera.frustum.fov;
+      return tweenInterval(
+        t => {
+          if (!(camera.frustum instanceof PerspectiveFrustum)) return;
+          camera.frustum.fov = (fov - fromFov) * t + fromFov;
+        },
+        easing || "inOutCubic",
+        (duration ?? 0) * 1000,
+      );
+    }
   }
   return undefined;
 };
 
 export const getCamera = (viewer: Viewer | CesiumWidget | undefined): Camera | undefined => {
   if (!viewer || viewer.isDestroyed() || !viewer.camera || !viewer.scene) return undefined;
   const { camera } = viewer;
-  if (!(camera.frustum instanceof PerspectiveFrustum)) return;
-
-  const ellipsoid = viewer.scene.globe.ellipsoid;
-  const { latitude, longitude, height } = ellipsoid.cartesianToCartographic(camera.position);
+  if (
+    !(
+      camera.frustum instanceof PerspectiveFrustum ||
+      camera.frustum instanceof OrthographicOffCenterFrustum
```

Comment:
Can you explain why we check if camera.frustum is an instanceof `OrthographicOffCenterFrustum` here? 
The PerspectiveFrustum makes sense, but not the OrthographicOffCenterFrustum. Does it make more sense for you to be checking if `camera.positionCartographic`, or am I missing something?

Reply:
The `camera.frustum` can be some other types which are not in use for the system by now. I checked this for returning expect result only when it is  `PerspectiveFrustum` for 3d/2.5d mode and `OrthographicOffCenterFrustum` for 2d mode.